packages/knip/src/plugins/sst/types.ts

```diff
@@ -0,0 +1,4 @@
+export type PluginConfig = {
+  plugins?: string[];
+  entryPathsOrPatterns?: string[];
```

Comment:
This is probably coming from the template, any chance it could reflect the actual SST config interface? If there's nothing interesting from the plugin perspective it could be left empty or even removed.